src/test/moves/obstruct.test.ts

```diff
@@ -43,6 +43,23 @@ describe("Moves - Obstruct", () => {
     expect(enemy.getStatStage(Stat.DEF)).toBe(-2);
   }, TIMEOUT);
 
+  it("obstruct miss test, should still reduce defense despite missed attack", async () => {
```

Comment:
```suggestion
  it("bypasses accuracy checks when applying protection and defense reduction", async () => {
```
Using the same naming/style conventions as the other tests in this file